### PR TITLE
chore(updatecli+lint) various non functionnal fixes

### PR DESCRIPTION
**Pull request by dduportal**
to ensure that latest hadolint accepts dockerfile + updatecli works with recent versions

Signed-off-by: Damien Duportal <damien.duportal@gmail.com>

### CODE REVIEW

**Comment by dduportal** [comment]
update diff fails, because egg/chicken issues (as usual):

```
ERROR: Something went wrong in target "updateReleaseInDockerfile" :

ERROR: ✗ No line found matching the keyword "ARG" and the matcher "tini_version".
```

File: updatecli/updatecli.d/mirrorbits.yaml [hunk 1]
```diff
@@ -1,13 +1,29 @@
+---
+title: "Bump blobxfer version"
```
Comment (lemeurherve) [inline updatecli/updatecli.d/mirrorbits.yaml]:
```suggestion
title: "Bump mirrorbits version"
```

File: updatecli/values.yaml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 github:
   user: "Jenkins Infra Bot (updatecli)"
   email: "60776566+jenkins-infra-bot@users.noreply.github.com"
+  username: "jenkins-infra-bot"
   token: "UPDATECLI_GITHUB_TOKEN"
-  branch: "master"
+  branch: "main"
   owner: "jenkins-infra"
-  repository: "docker-mirrorbits"
+  repository: "docker-builder"
```
Comment (lemeurherve) [inline updatecli/values.yaml]:
```suggestion
  repository: "docker-mirrorbits"
```

**Review by lemeurherve — CHANGES_REQUESTED** [review]

**Comment by dduportal** [comment]
@lemeurherve good catch! it explains why the updatecli failed in fact 😅 many thanks

**Review by lemeurherve — APPROVED** [review]

**Pull request merged**